src/lambda/vue/samInvokeVue.ts

```diff
@@ -287,39 +399,129 @@ export const Component = Vue.extend({
+                        <option v-for="(method, index) in httpMethods" v-bind:value="method.toLowerCase()" :key="index">
                             {{ method }}
                         </option>
-                    </select>
+                    </select><span class="data-view">{{launchConfig.api.httpMethod}}</span>
                 </div>
                 <div class="config-item">
                     <label for="query-string">Query String</label>
                     <input name="query-string" id="query-string" cols="15" rows="2" placeholder="Enter a query" />
                 </div>
                 <div class="config-item">
-                    <label for="">Headers</label>
-                    <button id="template-path-api-button">Add Header...</button>
+                    <label for="headers">Headers</label>
+                    <input type="text" v-model="headers.value" placeholder="Enter as valid JSON" >
+                    <div class="json-parse-error" v-if="headers.errorMsg">Error parsing JSON: {{headers.errorMsg}}</div>
                 </div>
             </div>
             <div v-else>Select an Invoke Target</div>
+            <button @click="toggleShowAllFields">{{showAllFields ? "Show Less Fields" : "Show All Fields"}}</button>
+            <div v-if="showAllFields">
+                <h3>aws</h3>
+                <div class="config-item">
+                    <label for="awsConnection">Credentials:</label>
+                    <input type="text" v-model="launchConfig.aws.credentials" >
+                </div>
+                <div class="config-item">
+                    <label for="region">Region</label>
+                    <input type="text" v-model="launchConfig.aws.region" >
+                </div>
+                <h3>lambda</h3>
+                <div class="config-item">
+                    <label for="">Environment Variables</label>
+                    <input type="text" placeholder="Enter as valid JSON" v-model="environmentVariables.value">
+                    <div class="json-parse-error" v-if="environmentVariables.errorMsg">Error parsing JSON: {{environmentVariables.errorMsg}}</div>
+                </div>
+                <div class="config-item">
+                    <label for="memory">Memory (MB)</label>
+                    <input type="number" v-model.number="launchConfig.lambda.memoryMb" >
+                </div>
+                <div class="config-item">
+                    <label for="timeoutSec">Timeout (s)</label>
+                    <input type="number" v-model.number="launchConfig.lambda.timeoutSec" >
+                </div>
+                <!-- <div class="config-item">
+                    <label for="pathMappings">Path Mappings</label>
+                    <input type="text" v-model="launchConfig.lambda.pathMappings" >
+                </div> -->
+                <h3>sam</h3>
+                <div class="config-item">
+                    <label for="buildArguments">Build Arguments</label>
```

Comment:
Maybe add a note for the array fields that a comma separated list of args should be provided.

Reply:
Added a placeholder: Enter as a comma separated list